ts/features/itwallet/screens/discovery/ItwDiscoveryInfoScreen.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
```

Comment:
What do you think about using a folder-by-feature instead of a folder-by-type structure? For example: `itwallet/discovery/screens` instead of `itwallet/screens/discovery`.
This way we could have everything related to itw discovery in the same folder.